3rdparty/CMakeLists.txt

```diff
@@ -144,4 +144,9 @@ if (IRR_BUILD_CEGUI)
 			PARENT_SCOPE
 		)
 	endif()
+
+	install(
+		DIRECTORY ${CEGUI_INCLUDE_PATH}/CEGUI
```

Comment:
do we need to expose CEGUI in the public headers of your `irr::ext::cegui` ?

Reply:
yes, because ext will only help you set up. In order to develop GUI, you'll have to use CEGUI API too.